common/math.go

```diff
@@ -8,6 +8,15 @@ import (
 	"math/rand"
 )
 
+func IntSliceContains(a []int, b) bool {
```

Comment:
No need to move it, but for future things I would keep the "numbers" package in mind.